### PR TITLE
1.4.0 to 1.5.0

### PR DESCRIPTION
**Pull request by vildead**
Tested on daisy-stable
Logs from upgrade are on Owncloud (Datastewardship_R3/DAISY_releases/1.5.0/)

Still requires:
- bump version
- create a tag
- add release notes (also on Owncloud)

Internal [policy](https://lcsb-wiki.uni.lu/xwiki/wiki/lcsbbiocore/view/Datastewardship/DAISY/) requires 1 datasteward and 1 developer to approve the request.

### CODE REVIEW

**Review by pinarpink — APPROVED** [review]
Big thanks @vildead

File: web/views/projects.py [hunk 1]
```diff
@@ -42,6 +43,8 @@ def project_list(request):
         'reset': True,
         'facets': facet_view_utils.filter_empty_facets(projects.facet_counts()),
         'query': query or '',
+        'filters': request.GET.get('filters') or '',
```
Comment (jLebioda) [inline web/views/projects.py:46]: maybe it should be `request.GET.get('filters', '')`?

File: web/views/datasets.py [hunk 2]
```diff
@@ -96,6 +97,8 @@ def dataset_list(request):
     )
     return render(request, 'search/search_page.html', {
         'reset': True,
+        'filters': request.GET.get('filters') or '',
```
Comment (jLebioda) [inline web/views/datasets.py:100]: maybe `request.GET.get('filters', '')`?

File: web/views/contracts.py [hunk 3]
```diff
@@ -95,13 +94,14 @@ def contract_list(request):
         'reset': True,
         'facets': facet_view_utils.filter_empty_facets(contracts.facet_counts()),
         'query': query or '',
+        'order_by': order_by or '',
+        'filters': request.GET.get('filters') or '',
```
Comment (jLebioda) [inline web/views/contracts.py:98]: maybe `request.GET.get('filters', '')`?

File: setup.py [hunk 4]
```diff
@@ -46,7 +50,7 @@
 
 setup(
     name='elixir-daisy',
-    version='1.4.0',
+    version='1.4.1',
```
Comment (jLebioda) [inline setup.py:53]: Probably the version could be already bumped

File: elixir_daisy/settings_local.template.py [hunk 5]
```diff
@@ -1,3 +1,5 @@
+GLOBAL_API_KEY = 'insert something random here'
+
```
Comment (jLebioda) [inline elixir_daisy/settings_local.template.py:2]: maybe we should do the same as in lines 13 and 14?

**Review by jLebioda — APPROVED** [review]
There are some minor suggestions from me

File: web/views/contracts.py (hunk 3 above)
Comment (vildead) [reply web/views/contracts.py]: ok, refactored

File: setup.py (hunk 4 above)
Comment (vildead) [reply setup.py]: version was bumped to 1.5.0

File: elixir_daisy/settings_local.template.py (hunk 5 above)
Comment (vildead) [reply elixir_daisy/settings_local.template.py:4]: resolved in 35a8291ac7246f59b663f712b1b70acd339f6196

File: web/views/projects.py (hunk 1 above)
Comment (vildead) [reply web/views/projects.py]: nice!
Refactored

**Pull request merged**